src/Microsoft.Management.Deployment/PackageManager.cpp

```diff
@@ -562,6 +567,10 @@ namespace winrt::Microsoft::Management::Deployment::implementation
     {
         // Add installed version
         AddInstalledVersionToContext(package.InstalledVersion(), comContext.get());
+
+        // Uninstall should not require agreements acceptance.

```

Comment:
Uninstall should never check for agreements, so this code is not needed

Reply:
Removed